topics:
  steering_command: /autodrive/f1tenth_1/steering_command
  throttle_command: /autodrive/f1tenth_1/throttle_command
  lidar: /autodrive/f1tenth_1/lidar
  debug_lidar: /ebva/debug/lidar
  steering_angle: /autodrive/f1tenth_1/steering

parameters:
  disparity_threshold: 0.05
  tol_mask_disparities: 0.01
  car_diameter: 0.85
  time_to_collision: 42
  min_speed_factor: 5
  safe_curve_exit_ratio: 0.333
  safe_lateral_distance: 0.1
  robustness_range: [85, 95]  # Considering angles near lateral beams (-90 and 90 degrees)

steering_controller:
  steering_saturation: 60 # degrees
  td: 0.2 # seconds
  kp_steer: 1
  n: 10 # number of points to consider
  ts: 0.025 # seconds
